esp/esdllib/esdl_def_helper.cpp

```diff
@@ -113,6 +113,58 @@ void EsdlDefinitionHelper::setTransformParams( EsdlXslTypeId xslId, IProperties
     parameters.setValue( xslId, params );
 }
 
+void removeEclHiddenStructs(IPropertyTree &depTree)
+{
+    Owned<IPropertyTreeIterator> it = depTree.getElements("*[@ecl_hide='1']");
```

Comment:
Out of curiosity, and not sure if this could ever happen but will getElements return attributes encapsulated in double quotes? "*[@ecl_hide=\"1"\]" ?